packages/contracts-bedrock/interfaces/safe/ILivenessModule2.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { ISemver } from "interfaces/universal/ISemver.sol";
+
+/// @title ILivenessModule2
+/// @notice Interface for LivenessModule2, a singleton module for challenge-based ownership transfer
+interface ILivenessModule2 is ISemver {
+
+    /// @notice Configuration for a Safe's liveness module
+    struct ModuleConfig {
+        uint256 livenessResponsePeriod;
+        address fallbackOwner;
+    }
+
+    /// @notice Error for when module is not enabled for the Safe
+    error LivenessModule2_ModuleNotEnabled();
+
+    /// @notice Error for when Safe is not configured for this module
+    error LivenessModule2_ModuleNotConfigured();
+
+    /// @notice Error for when a challenge already exists
+    error LivenessModule2_ChallengeAlreadyExists();
+
+    /// @notice Error for when no challenge exists
+    error LivenessModule2_ChallengeDoesNotExist();
+
+    /// @notice Error for when trying to cancel a challenge after the response period has ended
+    error LivenessModule2_ResponsePeriodEnded();
+
+    /// @notice Error for when trying to execute ownership transfer while the response period is still active
+    error LivenessModule2_ResponsePeriodActive();
+
+    /// @notice Error for when caller is not authorized
+    error LivenessModule2_UnauthorizedCaller();
+
+    /// @notice Error for invalid parameters
+    error LivenessModule2_InvalidParameters();
+
+    /// @notice Error for when trying to clear configuration while module is still enabled
+    error LivenessModule2_ModuleStillEnabled();
+
+    /// @notice Emitted when a Safe enables the module
+    event ModuleEnabled(address indexed safe, uint256 livenessResponsePeriod, address fallbackOwner);
+
+    /// @notice Emitted when a Safe disables the module
+    event ModuleDisabled(address indexed safe);
+
+    /// @notice Emitted when a challenge is started
+    event ChallengeStarted(address indexed safe, uint256 challengeStartTime);
+
+    /// @notice Emitted when a challenge is cancelled
+    event ChallengeCancelled(address indexed safe);
+
+    /// @notice Emitted when ownership is transferred to the fallback owner
+    event ChallengeExecuted(address indexed safe, address fallbackOwner);
+
+    /// @notice Reserved address used as the previous owner to the first owner in a Safe
+    /// @return The sentinel owner address (0x1)
+    function SENTINEL_OWNER() external view returns (address);
```

Comment:
We should try to avoid adding new SCREAMING_SNAKE_CASE names to the interface and instead do [this](https://github.com/ethereum-optimism/optimism/blob/2a4444e070f15d9dfde1d881690eb40d08d7e30d/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L298-L301).

Reply:
Sentinel owner dot need to be public, changing it into an internal constant

Reply:
Fixed in 669f2e16e6ceb96f1e2aeedaf8cd69e4f047f57a